src/cf-typography/src/atoms/links.less

```diff
@@ -0,0 +1,182 @@
+//
+// Links with icons
+//
+
+.a-link__icon {
+    border-bottom-width: 0;
+
+    .a-link_text {
+        border-bottom-width: 1px;
+        border-bottom-style: inherit;
+    }
+
+    &:before,
+    &:after {
+        .cf-icon();
+
+        margin-right: 1px;
+        margin-left: 1px;
+    }
+
+    &.a-link__download:after,
+    &.a-link__download&.a-link__before:before {
+        // Inserts the cf-icon-download glyph.
+        content: "\e406";
+    }
+
+    &.a-link__email:after,
+    &.a-link__email&.a-link__before:before {
+        // Inserts the cf-icon-email glyph.
+        content: "\e302";
+    }
+
+    &.a-link__external-link:after,
+    &.a-link__external-link&.a-link__before:before {
+        // Inserts the cf-icon-external-link glyph.
+        content: "\e610";
+    }
+
+    &.a-link__fax:after,
+    &.a-link__fax&.a-link__before:before {
+        // Inserts the cf-icon-fax glyph.
+        content: "\e310";
+    }
+
+    &.a-link__left:after,
+    &.a-link__left&.a-link__before:before {
+        // Inserts the cf-icon-left glyph.
+        content: "\e000";
+    }
+
+    &.a-link__mail:after,
+    &.a-link__mail&.a-link__before:before {
+        // Inserts the cf-icon-mail glyph.
+        content: "\e304";
+    }
+
+    &.a-link__pdf:after,
+    &.a-link__pdf&.a-link__before:before {
+        // Inserts the cf-icon-pdf glyph.
+        content: "\e402";
+    }
+
+    &.a-link__pdf:after,
+    &.a-link__pdf&.a-link__before:before {
+        // Inserts the cf-icon-pdf glyph.
+        content: "\e402";
+    }
+
+    &.a-link__phone:after,
+    &.a-link__phone&.a-link__before:before {
+        // Inserts the cf-icon-phone glyph.
+        content: "\e306";
+    }
+
+    &.a-link__right:after,
+    &.a-link__right&.a-link__before:before {
+        // Inserts the cf-icon-right glyph.
+        content: "\e002";
+    }
+
+    // New icon modifiers must be added ABOVE this line.
+
+    &.a-link__before {
+        &:after {
+            content: '';
+        }
+    }
+
+    &.a-link__no-wrap {
+        white-space: nowrap;
+    }
+}
+
+//
+// Styled link
+//
+
+.a-link__styled {
+    .webfont-medium();
+}
```

Comment:
I was hoping we could remove it and just use the medium font where we needed but wasn't sure. Consider it gone.